circle.yml

```diff
@@ -19,4 +19,5 @@ deployment:
   npm:
     tag: /^v[0-9]+(\.[0-9]+)*$/
     commands:
+      - make browser
```

Comment:
Please, move this to a `compile` section.

Reply:
pls, https://github.com/alissonperez/deepicker/pull/8/commits/76cc347d35041bbeca4a7ebf50d0f62f4ce7de40